spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe ApplicationHelper do
+  describe '#tooltip' do
+    it 'creates a div containing aria label with text and image' do
+      tooltip_text = 'foobar'
```

Comment:
There is a school of though that says you should avoid using locals like this in test code and just include the string `foobar` in each place.   Not sure how strictly we on this project want to conform to this but I test like this approach.